libraries/AP_HAL_ChibiOS/hwdef/f303-MatekGPS/hwdef-bl.dat

```diff
@@ -61,6 +61,7 @@ define PORT_INT_REQUIRED_STACK 64
 
 # avoid timer and RCIN threads to save memory
 define HAL_NO_TIMER_THREAD
+define HAL_NO_RCOUT_THREAD
```

Comment:
move all 3 of these defines for all bootloaders to chibios_hwdef.py

Reply:
Fixed